templates/definition/meter/openems.yaml

```diff
@@ -56,6 +56,21 @@ render: |
       user: x
       password: {{ .password }}
     jq: (.value // 0)
+  {{- if or (eq .usage "grid") (eq .usage "pv") }} 
```

Comment:
Könntest Du das bitte je usage auseinander ziehen? Unten ist ja schon ein if pv block.

Reply:
Schadet nichts

Reply:
Anders gesagt: wäre super, machen wir in anderen Templates auch so. Reihenfolge grid, pv, battery.